frontend/mobilityai_react/App/Screens/NewPatientForm.js

```diff
@@ -0,0 +1,294 @@
+import React from 'react';
+import {Platform, StyleSheet, Text, View, ScrollView, TextInput, Slider, Button, Picker} from 'react-native';
+
+import { CheckBox } from 'react-native-elements'
+import RadioForm, {RadioButton, RadioButtonInput, RadioButtonLabel} from 'react-native-simple-radio-button';
+
+export default class PatientForm extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            firstName: "",
+            lastName: "",
+            baselineWalk: 4,
+            baselineSit : 12,
+            baselineLay : 8,
+            baselineStand: 4,
+            conditionThatBroughtThem: "Orthopedic",
+            bodyPartsInvolved: {
+                neck: false,
+                middleBackRibs: false,
+                lowerBack: false,
+                shoulderArmOrElbow: false,
+                handOrWrist: false,
+                pelvisHipLegOrKnee: false,
+                footOrAnkle: false,
+                other: false,
+            },
+            onsetOfCondition: "",
+            previousSurgery: false,
+            severityOfCondition: "",
+            livingSituation: "",
+            walkingSituation: "",
+            difficultyWithBasicMobility: "",
+            difficultyWithDailyAcitity: "",
+        }
+    }
+
+    submitForm(){
+        console.log("TADA");
```

Comment:
Create an issue for integrating web requests from react native app to server.

Reply:
See #80